sdcm/sct_runner.py

```diff
@@ -447,7 +447,7 @@ class AwsSctRunner(SctRunner):
     SOURCE_IMAGE_REGION = "eu-west-2"  # where the source Runner image will be created and copied to other regions
     IMAGE_BUILDER_INSTANCE_TYPE = "t3.small"
     REGULAR_TEST_INSTANCE_TYPE = "t3.large"  # 2 vcpus, 8G, 36 CPU credits/hour
```

Comment:
Shouldn't we also change the regular instance type?
Reason: We get log lags probably caused by the SCT runner overload even on short longevities...

Reply:
me and @roydahan discussed it.

we'll put this for couple of week see the effect on tier1, and then we'll probably update this one as well

Reply:
I think you can change it from t3 anyway.
It can remain large, but change it to something more predictable than t3.

Reply:
r5.large for all ? I don't think it really matters for the test, it's just about price

we shouldn't be maxing it out (and now we could track it happing)

Reply:
m7i-flex.large instead of the t3 and m7i-flex.xlarge for the LONGTERM can stay.

Reply:
change it